src/social-network-provider/facebook.com/UI/injectPostBox.tsx

```diff
@@ -7,19 +8,45 @@ import { PostDialogHint } from '../../../components/InjectedComponents/PostDialo
 import { MessageCenter } from '../../../utils/messages'
 import { Flags } from '../../../utils/flags'
 
-let composeBox: LiveSelector<Element>
+interface UIProps {
+    commentElement?: Element
+    commentId?: string
+}
+
+let postEditorInPopupSelector: LiveSelector<Element>
 if (isMobileFacebook) {
-    composeBox = new LiveSelector().querySelector('#structured_composer_form')
+    postEditorInPopupSelector = new LiveSelector().querySelector('#structured_composer_form')
 } else {
-    composeBox = new LiveSelector()
+    postEditorInPopupSelector = new LiveSelector()
         .querySelectorAll('form [role="button"][tabindex="-1"]')
         .map((x) => x.parentElement)
         // TODO: should be nth(-1), see https://github.com/DimensionDev/Holoflows-Kit/issues/270
         .reverse()
         .nth(0)
 }
+
+const postEditorInCommentSelector: LiveSelector<Element> = new LiveSelector()
+    .querySelectorAll('[role="article"] [role="presentation"]')
+    .map((x) => x.parentElement)
+    .reverse()
+
 export function injectPostBoxFacebook() {
-    const watcher = new MutationObserverWatcher(composeBox.clone())
+    new MutationObserverWatcher(postEditorInCommentSelector.clone())
+        .useForeach((element, key, metadata) => {
+            const commentId = createHash('md5').update(element.innerHTML).digest().toString('hex')
```

Comment:
Maybe we should use a simple function to create a unique string, I tried `Date.now()` which does not work.